src/LaravelDebugbar.php

```diff
@@ -723,6 +723,7 @@ public function modifyResponse(Request $request, Response $response)
                 strpos($response->headers->get('Content-Type'), 'html') === false)
             || $request->getRequestFormat() !== 'html'
             || $response->getContent() === false
+            || $request->isXmlHttpRequest()
```

Comment:
Can you use `$this->isJsonRequest($request)` like above?

Reply:
sure